README.md

```diff
@@ -43,9 +43,13 @@ This **open-source plugin was developed to help the Sylius community**. If you h
 
 ## Installation
 ```bash
-$ composer require bitbag/blacklist-plugin
+composer require bitbag/blacklist-plugin -W
 ```
-    
+
+You will see error during cache:clear operation. It will disappear after whole installation process.
+
+![Origin](doc/images/installation-error.png)
```

Comment:
How about running the composer command with `--no-scripts` switch, and let user to run cache:clear at the end of process?

Reply:
Done. It's resolve issue